packages/sdk/src/Campaigns.ts

```diff
@@ -1,7 +1,10 @@
 import * as T from './types/Campaigns'
+import * as AAT from './types/AsyncActions'
 
 import type { RequestController } from './RequestController'
 import { encode } from './helpers'
+import * as fs from 'fs'
```

Comment:
Just wonder... this SDK can be used on the server and client side. I know that this specific endpoint I part of VoucherifyServerSide, but can attempting to use node specific library in our SDK somehow affect VoucherifyClientSide? Can we test it somehow between merge/release?

Reply:
![Screenshot 2023-09-13 at 17 58 32](https://github.com/p-zielinski/voucherify-js-sdk/assets/82354717/8c81069b-cead-43c4-a203-3b22841ba39d)
Tested with emporix project (project uses browser so does not have fs library) and works perfectly as long as we don't use csv import methods. We may add `try, catch` with a proper `error` if someone will try to use import csv function in browser environment.

Reply:
...
```
Module not found: Error: Can't resolve 'fs' in '/Users/piotrzielinski/Documents/GitHub/emporix-b2b-showcase/node_modules/@voucherify/sdk/dist'
```
You were right, I'll try to find a solution

Reply:
Added assertion, dynamic importing, and
```
"browser": {
		"fs": false,
		"os": false,
		"path": false
	}
```
solves the problem